cmd/xyz2svg/svg_files/.gitignore

```diff
@@ -1 +0,0 @@
-*.svg
```

Comment:
Why are we removing the *.svg files from here. We don't want to accidentally check in svg files.